src/util/typeHelper.ts

```diff
@@ -135,6 +135,15 @@ export class Helper{
             hour: 'numeric', minute: 'numeric', second: 'numeric'
           }).format(date);
       }
+
+      static convertDisplayDateTimeFormatShort(dateTimeJSON: string): string {
```

Comment:
still named as short ?

Reply:
Sorry, likely it is in another branch. Will change accordingly asap, to `convertDisplayDateTimeFormat24`

Reply:
ok